<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pontszamModositasa</class>
 <widget class="QMainWindow" name="pontszamModositasa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pontszám modosítás</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/img/quizzyLogo.png</normaloff>:/resource/img/quizzyLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Wingdings</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pontszám modosítása</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="felhasznaloNevLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Felhasználó név</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modositottPontszamLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Módosított pontszám</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mehetGomb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Mehet</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="temakorLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Témakör</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>felhasznaloNevLineEdit</tabstop>
  <tabstop>temakorLineEdit</tabstop>
  <tabstop>modositottPontszamLineEdit</tabstop>
  <tabstop>mehetGomb</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
